quotas/quotus.js

```diff
@@ -40,10 +40,54 @@ function ReadQuotaTables() {
             data += qBuff.childNodes[i].childNodes[0].innerText + "\n";
         }
     }
-    data = data.split("\n");
-    console.log(data);
+    //data = data.split("\n");
```

Comment:
Here don't change the function logic. Some functions are looking for the data to be split by new line. Rather in your function:
```javascript
let myData = ReadQuotaTables().join("")
```
for the same behaviour.

Reply:
OK- I wasn't aware of anything else using it because the return wasn't there, that's my bad